Articles/Section-Trees/trees.md

```diff
@@ -0,0 +1,401 @@
+# Trees
+
+This section describes how to work with and create trees with Umbraco APIs.
+
+## Creating trees
+
+To Create a Tree in a section of the Umbraco backoffice, you need to take several steps:
+
+Create a 'TreeController' class in C#. A new controller which inherits from the abstract *Umbraco.Cms.Web.BackOffice.Trees.TreeController* class and provides an implementation for two abstract methods:
+
+* GetTreeNodes (returns a *TreeNodeCollection*) - Responsible for rendering the content of the tree structure;
+* GetMenuForNode (returns a *MenuItemCollection*) - Responsible for returning the menu structure to use for a particular node within a tree.
+
+Decorate your '*TreeController*' with the *Tree* Attribute, which is used to define the name of the section the Tree should be loaded in, which 'Tree Group' it should belong to and also define an alias and title for your custom tree.
+
+
+**For example**
+
+```csharp
+[Tree("settings", "favouriteThingsAlias", TreeTitle = "Favourite Things Name", TreeGroup="favouritesGroup", SortOrder=5)]
+public class FavouriteThingsTreeController : TreeController
+{
```

Comment:
Looks like the formatting in the rest of this article is a little wonky, and I suspect that the unclosed brackets here might be the culprit 🤔 

```suggestion
{ }
```